src/wire/tcp.rs

```diff
@@ -806,9 +819,30 @@ pub struct Repr<'a> {
     pub max_seg_size: Option<u16>,
     pub sack_permitted: bool,
     pub sack_ranges: [Option<(u32, u32)>; 3],
+    pub timestamp: Option<TcpTimestampRepr>,
     pub payload: &'a [u8],
 }
 
+#[derive(Debug, PartialEq, Eq, Clone, Copy)]
+pub struct TcpTimestampRepr {
+    tsval: u32,
```

Comment:
These fields should be public, since the accessors don't have any added behavior; all of the defined inherent methods don't bring anything over normal field access.